### PR TITLE
get_url fix word emphasis on documentation

### LINKED ISSUE #77241: get_url documentation formats "not" incorrectly
### Summary

The `get_url` module documentation has extraneous formatting in the "not" word in:
```
NTLM authentication is C(not) supported even if the GSSAPI mech for NTLM has been installed.
```
As it is a part of a sentence, I suspect the "not" here should not have any specific formatting.

### Issue Type

Documentation Report

### Component Name

get_url

### Ansible Version

```console
$ ansible --version
irrelevant
```

### Configuration

```console
$ ansible-config dump --only-changed
irrelevant
```

### OS / Environment

irrelevant

### Additional Information

irrelevant

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by bluikko**
##### SUMMARY

Fixes #77241 

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

get_url

##### ADDITIONAL INFORMATION

Change strong word emphasis from `C()` to `I()` which is used at other places of the same documentation for the same purpose.

```paste below

```

### CODE REVIEW

File: changelogs/fragments/77247-get_url-doc-emphasis.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
```
Comment (samccann) [inline changelogs/fragments/77247-get_url-doc-emphasis.yml]:
```suggestion
trivial:
```

**Review by samccann — APPROVED** [review]
Thanks for the PR! We don't require a changelog fragment for docs changes, so I just removed that file.

**Comment by samccann** [comment]
Thanks @bluikko for the issue and PR to fix it!  We talk about Ansible documentation on maxtrix at [#docs:ansible.im](https://matrix.to/#/#docs:ansible.im) and on libera IRC at #ansible-docs if you ever want to join us and chat about the docs!

File: changelogs/fragments/77247-get_url-doc-emphasis.yml (hunk 1 above)
Comment (bluikko) [reply changelogs/fragments/77247-get_url-doc-emphasis.yml]: What is this `trivial` category? It is not listed in https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment so I did not know it even exists!

Comment (samccann) [reply changelogs/fragments/77247-get_url-doc-emphasis.yml]: Huh.. did not realize it wasn't documented. Some collection owners use it when they require a changelog fragment for every PR. It's here - https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs.  `trivial` fragments won't appear in the changelog file.  I attempted that change until someone told me I could delete the file here in the PR.
I'll have to ask and see if it works for core changelogs as well...

**Pull request merged**